my_tinder/tests/test_views.py

```diff
@@ -66,53 +69,74 @@ def use_dummy_cache_backend(settings):
 
 
 @pytest.fixture()
-def one_user():
+def one_user(db):
```

Comment:
fixtures must sit in a separate file. See how it is done here: https://github.com/thenewboston-blockchain/Node/tree/master/node/blockchain/tests/fixtures